ajax/libs/highlighter.js/0.1.8/highlighter.min.js

```diff
@@ -0,0 +1,12 @@
+//# sourceMappingURL=highlighter.min.js.map
```

Comment:
@PeterDaveHello @kennynaoh The upstream has map file. I'm not sure if it will have any effect.